workload/run.yml

```diff
@@ -11,10 +11,12 @@ rally:
       times: 
         - 10
     scenarios:
-      - name: boot and list servers 
-        file: nova-boot-list-cc.yml
-        args: 
-          sla_max_avg_duration: 30
+    - name: boot and list servers
+      file: scenarios/nova-boot-list-cc.yml
```

Comment:
Since the workload directory is only an example, I would prefer not to create a subdirectory only for one file.